### PR TITLE
fix: maintain all row spans after editing some cell

### PR DESCRIPTION
**Pull request by jajugoguma**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x]Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

* Fixed an issue that the row spans released after editing some cell.
  * It occurs by resetting row spans regardless of using dynamic row span.
  * Changed to reset row spans when only using dynamic row span.

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: packages/toast-ui.grid/src/dispatch/rowSpan.ts [hunk 1]
```diff
@@ -138,14 +138,16 @@ function updateSubRowSpan(
   }
 }
 
-export function resetRowSpan({ data }: Store, slient = false) {
-  data.rawData.forEach(({ rowSpanMap }) => {
-    Object.keys(rowSpanMap).forEach((columnName) => {
-      delete rowSpanMap[columnName];
+export function resetRowSpan({ data, column }: Store, slient = false) {
+  if (column.visibleRowSpanEnabledColumns.length > 0) {
```
Comment (adhrinae) [inline packages/toast-ui.grid/src/dispatch/rowSpan.ts:142]: 저라면 들여쓰기를 한 번 더 하느니 early return을 할 것 같은데 다른 분들의 생각이 궁금하군요.

**Review by adhrinae — APPROVED** [review]
LGTM

**Review by lja1018 — APPROVED** [review]
리뷰 완료합니다.

File: packages/toast-ui.grid/src/dispatch/rowSpan.ts (hunk 1 above)
Comment (lja1018) [reply packages/toast-ui.grid/src/dispatch/rowSpan.ts:142]: 생각해보니 이정도는 early return이 깔끔할 것 같네요.

**Review by dotaitch — APPROVED** [review]

**Review by jwlee1108 — APPROVED** [review]
리뷰 완료합니다.

**Review by adhrinae — APPROVED** [review]

**Pull request merged**